.changeset/unlucky-cooks-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+---
+
+[New Functionality] Support fetching all destinations from the destination service simultaneously.
```

Comment:
[req] Be a bit more precise like: All destinations of a subacction and service instance simultaneously.